internal/ignition/installmanifests.go

```diff
@@ -1317,6 +1322,25 @@ func bmhIsMaster(bmh *bmh_v1alpha1.BareMetalHost, masterHostnames, workerHostnam
 	return strings.Contains(bmh.Name, "-master-")
 }
 
+// For the agent-installer, skip files which have been regenerated.
+func skipAgentUpload(ctx context.Context, key string, s3Client s3wrapper.API) bool {
+	skipFiles := []string{
+		"kubeadmin-password",
+		"99_kubeadmin-password-secret.yaml",
```

Comment:
From a quick test the console login is still broken. The password update is skipped, but it seems that `"99_kubeadmin-password-secret.yaml"` is generated before (so this skip is ignored here essentially), so very likely it's stil being generated with the new kubeadmin-password.hash